lighthouse-cli/test/smokehouse/test-definitions/issues/mixed-content.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
moved from `dbw-expectations.js` as dbw is kind of its own thing and issues are worthy of their own smoke tests